llvm/test/FileCheck/empty-variable-name.txt

```diff
@@ -0,0 +1,32 @@
+a
+
+; RUN: not FileCheck -input-file %s %s 2>&1 | \
+; RUN:   FileCheck -check-prefix CHECK-ERROR -DDIR=%S \
+; RUN:   --match-full-lines --strict-whitespace %s
+
+; CHECK:        a[[]]
+; CHECK-ERROR:[[DIR]]/empty-variable-name.txt:7:13: error: empty variable name
+; CHECK-ERROR-NEXT:; CHECK: a{{\[\[\]\]}}
```

Comment:
What we do typically is add some space before the CHECK directive so that the caret (^) aligns with the caracter it points to in the real output. So for instance the output here is:
```
; CHECK: a[[]]
            ^
```

with the caret just under the `]` so you would write:

```
; CHECK-ERROR-NEXT:; CHECK: a{{\[\[\]\]}}
;      CHECK-ERROR-NEXT:            ^
```

Reply:
Fixed, thanks, see 6974c787759218651f04a67c6e4bc5de76d80999.